src/transformers/generation_utils.py

```diff
@@ -39,6 +40,69 @@
 logger = logging.get_logger(__name__)
 
 
+@dataclass
+class GreedySearchDecoderOnlyOutput(ModelOutput):
+    """
+    Base class for outputs of encoder-decoder generation models using greedy search.
+
+    Args:
+        sequences (:obj: `torch.LongTensor` of shape :obj:`(batch_size * num_return_sequences, sequence_length)`):
+            The generated sequences. The second dimension (sequence_length) is either equal to :obj:`max_length` or shorter if all
+            batches finished early due to the :obj:`eos_token_id`.
+        logits (:obj:`torch.FloatTensor` of shape :ob:`(batch_size * num_return_sequence, sequence_length, config.vocab_size)`, `optional`, returned when ``output_scores=True`` is passed or when ``config.output_scores=True``):
```

Comment:
``
```suggestion
        logits (:obj:`tuple(torch.FloatTensor)`, `optional`, returned when ``output_scores=True`` is passed or when ``config.output_scores=True``):
        Tuple of :obj:`torch.FloatTensor` of length :obj:`max_length` or shorter if all batches finished early, with each tensor of shape :obj:`(batch_size, config.vocab_size)`).
```

Reply:
think each logit only has shape `(batch_size, config.vocab_size)` => but to check